storage/mysql/storage.sql

```diff
@@ -22,8 +22,8 @@ CREATE TABLE IF NOT EXISTS Trees(
   Description           VARCHAR(200),
   CreateTimeMillis      BIGINT NOT NULL,
   UpdateTimeMillis      BIGINT NOT NULL,
-  PrivateKey            BLOB NOT NULL,
-  PublicKey             BLOB NOT NULL,
+  PrivateKey            MEDIUMBLOB NOT NULL,
```

Comment:
I *think* BLOB is good enough for both Private and Public keys. Private are serialized protos (currently containing a file path), publics are DER-encoded keys (but not certs).